tracer/src/Datadog.Trace/ClrProfiler/NativeMethods.cs

```diff
@@ -59,42 +59,6 @@ public static void RemoveCallTargetDefinitions(string id, NativeCallTargetDefini
             }
         }
 
-        public static void EnableByRefInstrumentation()
-        {
-            if (IsWindows)
-            {
-                Windows.EnableByRefInstrumentation();
-            }
-            else
-            {
-                NonWindows.EnableByRefInstrumentation();
-            }
-        }
-
-        public static void EnableCallTargetStateByRef()
-        {
-            if (IsWindows)
-            {
-                Windows.EnableCallTargetStateByRef();
-            }
-            else
-            {
-                NonWindows.EnableCallTargetStateByRef();
-            }
-        }
-
-        public static void DisableTracerCLRProfiler()
```

Comment:
so we don't need this anymore? I still see it in the Native.def exports though ( same for the ones above I think)